gr-data.csv

```diff
@@ -1,4 +1,5 @@
 Project URL,SHA Detected,Module Path,Fully-Qualified Test Name (packageName.ClassName.methodName),Category,Status,PR Link,Notes
+
```

Comment:
Remove this empty line (and maybe even update the checker to automatically check for such mistakes).